src/domain/payments/index.types.d.ts

```diff
@@ -1,11 +1,27 @@
-type PriceRatio = {
+type PriceRatio<S extends WalletCurrency> = {
+  convertFromOther(other: bigint): PaymentAmount<S>
+  convertFromWallet(btcWalletAmount: PaymentAmount<S>): bigint
+  convertFromWalletToFloor(btcWalletAmount: PaymentAmount<S>): bigint
+  convertFromWalletToCeil(btcWalletAmount: PaymentAmount<S>): bigint
+  otherUnitPerWalletUnit(): DisplayCurrencyBasePerSat
```

Comment:
- not sure if should be responsability of this Floor/Ceil 
- `FromWallet` but param is `PaymentAmount` should we use `convertFromPaymentAmount`?
- `Other` is a bit confusing... `convertFromBigInt` or `convertToPaymentAmount`? :thinking:

Reply:
> * `FromWallet` but param is `PaymentAmount` should we use `convertFromPaymentAmount`?

Planning to rename `PaymentAmount` to `WalletAmount` in a follow-up PR